app/src/main/java/by/zharikov/myapplication/homework3/Homework3.kt

```diff
@@ -0,0 +1,156 @@
+package by.zharikov.myapplication.homework3
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import by.zharikov.myapplication.R
+import kotlinx.android.synthetic.main.activity_homework3.*
+
+class Homework3 : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_homework3)
+        val brestViewModel = ViewModelProvider(this).get(BrestViewModel::class.java)
+        brestViewModel.counterPotatoes.observe(this, Observer {
+            textView.text = it.toString()
+        })
+        brestViewModel.counterCabbage.observe(this, Observer {
+            textView2.text = it.toString()
+        })
+        brestViewModel.counterBeet.observe(this, Observer {
+            textView3.text = it.toString()
+        })
+
+        val vitebskViewModel = ViewModelProvider(this).get(VitebskViewModel::class.java)
```

Comment:
I thing you don't need class for each country bc your parent class do all what you need

Reply:
+

Reply:
I agree